BUILD.bazel

```diff
@@ -20,117 +20,126 @@ exports_files([
-    ],
-)
-
-cc_library(
-    name = "experimental-apigateway",
-    deps = [
-        "//google/cloud/apigateway:google_cloud_cpp_apigateway",
-    ],
-)
-
-cc_library(
-    name = "experimental-apigateway_mocks",
-    deps = [
-        "//google/cloud/apigateway:google_cloud_cpp_apigateway_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-accesscontextmanager",
-    deps = [
-        "//google/cloud/accesscontextmanager:google_cloud_cpp_accesscontextmanager",
-    ],
-)
-
-cc_library(
-    name = "experimental-accesscontextmanager_mocks",
-    deps = [
-        "//google/cloud/accesscontextmanager:google_cloud_cpp_accesscontextmanager_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-appengine",
-    deps = [
-        "//google/cloud/appengine:google_cloud_cpp_appengine",
-    ],
-)
-
-cc_library(
-    name = "experimental-appengine_mocks",
-    deps = [
-        "//google/cloud/appengine:google_cloud_cpp_appengine_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-artifactregistry",
-    deps = [
-        "//google/cloud/artifactregistry:google_cloud_cpp_artifactregistry",
-    ],
-)
-
-cc_library(
-    name = "experimental-artifactregistry_mocks",
-    deps = [
-        "//google/cloud/artifactregistry:google_cloud_cpp_artifactregistry_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-asset",
-    deps = [
-        "//google/cloud/asset:google_cloud_cpp_asset",
-    ],
-)
-
-cc_library(
-    name = "experimental-asset_mocks",
-    deps = [
-        "//google/cloud/asset:google_cloud_cpp_asset_mocks",
-    ],
-)
-
-cc_library(
-    name = "experimental-assuredworkloads",
+EXPERIMENTAL_LIBRARIES = [
+    "accessapproval",
+    "accesscontextmanager",
+    "apigateway",
+    "appengine",
+    "artifactregistry",
+    "asset",
+    "assuredworkloads",
+    "automl",
+    "billing",
+    "binaryauthorization",
+    "channel",
+    "cloudbuild",
+    "composer",
+    "container",
+    "containeranalysis",
+    "datacatalog",
+    "datamigration",
+    "debugger",
+    "dlp",
+    "eventarc",
+    "filestore",
+    "functions",
+    "gameservices",
+    "gkehub",
+    "iap",
+    "ids",
+    "iot",
+    "kms",
+    "logging",
+    "managedidentities",
+    "memcache",
+    "monitoring",
+    "networkmanagement",
+    "notebooks",
+    "orgpolicy",
+    "osconfig",
+    "oslogin",
+    "policytroubleshooter",
+    "privateca",
+    "pubsublite",
+    "recommender",
+    "redis",
+    "resourcemanager",
+    "retail",
+    "scheduler",
+    "securitycenter",
+    "servicecontrol",
+    "servicedirectory",
+    "servicemanagement",
+    "serviceusage",
+    "shell",
+    "storagetransfer",
+    "talent",
+    "texttospeech",
+    "tpu",
+    "trace",
+    "translate",
+    "videointelligence",
+    "vision",
+    "vmmigration",
+    "vpcaccess",
+    "webrisk",
+    "websecurityscanner",
+    "workflows",
+]
+
+TRANSITION_LIBRARIES = [
+    "secretmanager",
+    "tasks",
+]
+
+GA_LIBRARIES = [
+]
```

Comment:
do you want to add `bigquery`, `iam`, `pubsub`, `spanner` to here?

(leaving their deprecated targets below)

Reply:
I am leaving those for a later refactoring.